src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -453,6 +453,11 @@ export const LoadedRow = forwardRef((props: LoadedRowProps, ref: ForwardedRef<HT
     search_token_address_input: filterString,
   }
 
+  const price =
+    token.market?.price?.value !== undefined
+      ? formatUSDPrice(token.market?.price?.value, NumberType.FiatTokenStats)
```

Comment:
doesnt conedison already handle this for you? https://github.com/Uniswap/conedison/blob/main/src/format.ts#L366

Reply:
sure looks like it should--I'll look into why it wasn't doing that